Code/Tools/Libs/GuiFoundation/NodeEditor/NodeScene.moc.h

```diff
@@ -43,8 +43,9 @@ class EZ_GUIFOUNDATION_DLL ezQtNodeScene : public QGraphicsScene
     {
       BezierCurve,
       StraightLine,
+      SubwayLines,
 
-      Default = BezierCurve
+      Default = SubwayLines
```

Comment:
I don't think we should change the default. There should be a view option or something like that where the user can toggle between bezier and subway lines.

Reply:
I feel like ideally it should be something in the editor settings along with path width and other things like that. But i havent looked into doing that kind of thing yet. The change there was more for testing and can be reverted but i feel it gives a much more polished look for now until there is a setting for it.

Reply:
I agree that it looks nice. The reason I'm hesitant to change the default is that you will position your nodes depending on how the connections behave to untangle the mess a bit. If we simply change the default existing graphs might look weird because they have been layouted with a different connection behavior. I will play around with these changes over the weekend and test them on a few more complex graphs.

Reply:
I gave it a shot with a couple of complex graphs. It is actually not too bad, but I would still not change the default as it can be confusing in some existing graphs. I can look into adding the corresponding editor setting after this PR has been merged.